web/src/features/hfiCalculator/components/LastUpdatedHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import { StationDaily } from 'api/hfiCalculatorAPI'
+import React from 'react'
+import UpdateIcon from '@material-ui/icons/Update'
+import { makeStyles } from '@material-ui/core'
+
+export interface LastUpdatedHeaderProps {
+  dailies: StationDaily[]
+}
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    height: '30px'
+  },
+  icon: {
+    marginTop: '5px',
+    marginLeft: '5px'
+  },
+  headerText: {
+    marginLeft: '5px',
+    marginTop: '6px'
+  }
+})
+
+const findLastUpdate = (dailies: StationDaily[]) => {
+  let lastUpdated: any = undefined
+  dailies.forEach(daily => {
+    if (!lastUpdated || daily.last_updated > lastUpdated) {
+      lastUpdated = daily.last_updated
+    }
+  })
+  return lastUpdated
+}
+
+const LastUpdatedHeader = (props: LastUpdatedHeaderProps) => {
+  const classes = useStyles()
+  const lastUpdate = findLastUpdate(props.dailies)
+  if (lastUpdate) {
+    const dateString = new Intl.DateTimeFormat('en', {
+      month: 'long',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: 'numeric',
+      hour12: false,
+      timeZoneName: 'short'
+    })
+      .format(new Date(lastUpdate))
+      .toString()
+
+    return (
+      <React.Fragment>
+        <span className={classes.container}>
+          <UpdateIcon className={classes.icon}></UpdateIcon>
+          <p className={classes.headerText}>Forecast last updated {dateString}</p>
+        </span>
+      </React.Fragment>
+    )
+  } else {
+    return (
+      <React.Fragment>
+        <p>Could not find last updated station</p>
```

Comment:
Is this technically correct? There's no update date - should this read something like "Could not find last updated date". Also I'm wondering - maybe a question for @tessmgood , but is there even any value in pushing this to the user? Maybe it should just log the error to the console, I just don't see a user being able to take any action on this.